### PR TITLE
Create changelog for 0.19.1

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Review by zimlu02 — APPROVED** [review]

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+#### Changed
+* Error message for invalid variable definition
+* Analysis exceptions propagated to Extended API responses
+
+#### Fixed
+* IDMS statements in conditional blocks
+* Access to undefined element if there is no workspace open
+* Error on multiline optional paragraphs
+* Hover for group items
+* Loading copybooks with any encoding
```
Comment (zeibura) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:16]: What does this one actually mean? All encoding types are now supported for Copybooks? Isn't that more a new feature than a fix?

Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:16]: There was a bug that copybooks in one-byte national encodings throw an exception while loading, so this is rather a fix.

**Comment by temanbrcom** [comment]
Should be squashed on merge

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:16]: In that case I'd change it to "Error when loading copybooks with one-byte national encoding" - write what's been fixed.

Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:16]: That makes sense

**Review by zeibura — APPROVED** [review]

**Comment by temanbrcom** [comment]
@zeibura please, check the last lines in the "fix" sections

**Pull request merged**